operators.js

```diff
@@ -101,8 +96,8 @@ function debug() {
 function slowEqual(seekDesc, target, opts) {
   opts = opts || {}
   const seek = seekFromDesc(seekDesc)
-  const value = toBufferOrFalsy(target)
-  const valueName = !value ? '' : value.toString()
+  const value = bipf.allocAndEncode(target)
+  const valueName = !target ? '' : `${target}`
```

Comment:
We need to be really careful with these so that the indexes has the exact same names as before. The tests in ssb-db2 actually failed without this change :)